hack/jenkins/upload-to-gcs.sh

```diff
@@ -151,15 +151,22 @@ function update_job_result_cache() {
 
   mkdir -p ${tmp_results%/*}
 
+  # Construct a valid json file
+  echo "{\"JOB\" : [" > ${tmp_results}
+
   for upload_attempt in $(seq 3); do
     echo "Copying ${job_results} to ${tmp_results} (attempt ${upload_attempt})"
-    gsutil -q cp ${job_results} ${tmp_results} 2>&- || continue
+    gsutil -q cat ${job_results} 2>&- |\
+     sed -n 's/^\({"version".*}\),*/\1,/p' >> ${tmp_results} || continue
```

Comment:
is this doing the clever thing of removing all the wrapping each time?

(ugh we really need to be able to use `jq` here.) @spxtr should we just create an upload-to-gcs docker image with gcloud and jq installed?